shell/common/vsync_waiter_fallback.h

```diff
@@ -17,10 +17,15 @@ class VsyncWaiterFallback final : public VsyncWaiter {
  public:
   VsyncWaiterFallback(TaskRunners task_runners);
 
+  VsyncWaiterFallback(TaskRunners task_runners, float display_refresh_rate);
```

Comment:
Update the class doc comment to cover this ctor.

Reply:
Done